gcp-yaml/Pulumi.yaml

```diff
@@ -10,11 +10,11 @@ template:
 
 resources:
   # Create a GCP resource (Storage Bucket)
-  bucket:
+  my-bucket:
```

Comment:
Happened to notice these as well, so fixing them up, too. They aren't causing a problem, as the .append file renames them as `my-bucket`, but I can't think of a reason not to keep them in sync.